README.md

```diff
@@ -1,26 +1,40 @@
-- Name feature branches `feature_descriptive-name`
-- Name bug fix branches `fix_descriptive-name` 
-- Merge branches into __dev__ branch to deploy to development environment
-- Do not delete your feature branch until it is merged to production 
-- Keep your feature branch up to date with __main__
-- Merge feature branches into __main__ branch to deploy to production environment
-- After a production deployment, an admin needs to update __dev__ environment by resetting it to be up to date with __main__. Developers with features currently merged to dev env but not yet deployed to production should be advised to merge their features back into dev if they need to continue testing them in dev environment. 
-  - Navigate to `dev` branch in the terminal and run `git reset --hard origin/main`
-  - Force push dev branch `git push origin dev --force` This will trigger a new build of the `dev` branch to be deployed to the dev env.
-- When deploying to production, please create a corresponding Release in github ([Managing Releases in A Repository](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)). Each release PR should be   titled with it's release number using the [Semantic Versioning](https://semver.org/) convention (v1.0.0). 
+- Name feature branches `feature_descriptive-name`
+- Name bug fix branches `fix_descriptive-name`
 
+### When you are ready to test your code in a non-prod/ development environment
 
+- Merge branches into `dev` branch to deploy to non-prod/ development environment https://app.pennsieve.net/ - we only have one non-prod/ development environment at this time.
+- NOTE: Do not delete your feature branch until it is merged to production
+- Keep your feature branch up to date with `main.`
+
+### When you are ready to deploy your code to production
+
+- Make a PR from your feature branch into `main.`
+- It is a good idea to test this locally first before making your PR. Merge your code into `main` locally and test it locally.
+- You will need one approval to merge your code to `main`. Once you merge, it will automatically deploy to production https://app.pennsieve.io/
+- If you need to wait to deploy after PR approval, add a label to your PR `Do not merge` and another label describing why, such as `BE Dependency` or `Waiting for PM Approval` until you are ready to deploy, and then remove the labels. You can add a `Ready to Deploy` label to indicate it is ready to go out if you are waiting for someone else to deploy it, but feel free to merge it yourself if it is ready to go out, there is no formal process preventing any developer from deploying.
+- Merge feature branches into main branch to deploy to production environment.
+- When deploying to production, please create a corresponding Release in github (Managing Releases in A Repository). Each release PR should be titled with it's release number using the Semantic Versioning convention (v1.0.0).
```

Comment:
The steps are very clear, looking great. I have minor questions while creating a new tag/release, appreciate your thoughts on these
1. Should we always 'create a new tag' or is there any scenario we would use an existing tag?
2. Is creating a tag/release anyway tied up to deployment pipeline? 
3. What data is 'mandatory' in release notes, is 'Generate release notes' being used?

Also, Noticed this link to 'Managing Releases in A Repository' is missing